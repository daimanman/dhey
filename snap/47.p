{"C":1,"Duration":63670410003,"EndTime":"2018-08-21T09:00:03.3515864+08:00","Err":"","FileId":"4","Method":"POST","N":1,"P":{"info":"TEsss颠三倒四ST","name":"DXwwM"},"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":"2018-08-21T09:00:02.9485161+08:00","Status":1,"TaskId":47,"Type":"FIXED","Url":"http://localhost:9188/go1","Z":""}