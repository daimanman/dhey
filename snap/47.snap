{"AvgTotal":0.4020682,"Fastest":0.4020682,"Slowest":0.4020682,"Average":0.4020682,"Rps":2.4871402413819346,"AvgConn":0.0010032,"AvgDNS":0,"AvgReq":0,"AvgRes":0,"AvgDelay":0.401065,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0,"ReqMax":0,"ResMin":0,"ResMax":0,"DelayMax":0.401065,"DelayMin":0.401065,"Lats":[0.4020682],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0],"ResLats":[0],"DelayLats":[0.401065],"Total":402068200,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":45,"SizeReq":45,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.4020682,"Count":1,"Frequentcy":1},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0},{"Mark":0.4020682,"Count":0,"Frequentcy":0}],"TaskId":47,"Duration":0.4020682}